### PR TITLE
[Engine.Analyze] ClusteringEngine: set correct values in load()

### PR DESCRIPTION
**Pull request by fredroy**
Trying to implement strict type for setValue() and stumbled on ClusteringEngine which was setting wrong values (copy-paste I suppose)

\+ fix other strict type usages

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Comment by fredroy** [comment]
[ci-build][with-all-tests]

File: Sofa/Component/LinearSolver/Preconditioner/src/sofa/component/linearsolver/preconditioner/PrecomputedWarpPreconditioner.inl [hunk 1]
```diff
@@ -344,16 +344,16 @@ void PrecomputedWarpPreconditioner<TDataTypes>::loadMatrixWithSolver()
...
-        buf_tolerance = (double) CGlinearSolver->d_tolerance.getValue();
-        buf_maxIter   = (int) CGlinearSolver->d_maxIter.getValue();
-        buf_threshold = (double) CGlinearSolver->d_smallDenominatorThreshold.getValue();
-        CGlinearSolver->d_tolerance.setValue(1e-35);
-        CGlinearSolver->d_maxIter.setValue(5000);
-        CGlinearSolver->d_smallDenominatorThreshold.setValue(1e-25);
+        buf_tolerance = CGlinearSolver->d_tolerance.getValue();
+        buf_maxIter   = CGlinearSolver->d_maxIter.getValue();
+        buf_threshold = CGlinearSolver->d_smallDenominatorThreshold.getValue();
+        CGlinearSolver->d_tolerance.setValue(Real(1e-35));
```
Comment (alxbilger) [inline Sofa/Component/LinearSolver/Preconditioner/src/sofa/component/linearsolver/preconditioner/PrecomputedWarpPreconditioner.inl:354]: Not in the PR but 1e-35 is way too small, even for a double precision.

File: Sofa/GUI/Qt/src/sofa/gui/qt/viewer/qt/QtViewer.cpp [hunk 2]
```diff
@@ -932,8 +932,8 @@ void QtViewer::calcProjection(int width, int height)
         currentCamera->setBoundingBox(vparams->sceneBBox().minBBox(), vparams->sceneBBox().maxBBox());
     }
     currentCamera->computeZ();
-    currentCamera->p_widthViewport.setValue(width);
-    currentCamera->p_heightViewport.setValue(height);
+    currentCamera->p_widthViewport.setValue(sofa::Size(width));
```
Comment (damienmarchal) [inline Sofa/GUI/Qt/src/sofa/gui/qt/viewer/qt/QtViewer.cpp:935]: as we are now using user defined litteral for _sreal... maybe we could have others for _size and _index ?

Comment (alxbilger) [reply Sofa/GUI/Qt/src/sofa/gui/qt/viewer/qt/QtViewer.cpp:935]: Sure, but litterals applies to constants, which is not the case here.

**Review by alxbilger — APPROVED** [review]

**Pull request merged**